.env.example

```
@@ -37,8 +37,9 @@ IGDB_CLIENT_SECRET=your_igdb_client_secret_here
 TAVILY_API_KEY=your_tavily_api_key_here
 
 # Supabase Configuration (when needed)
-# SUPABASE_URL=your_supabase_url
-# SUPABASE_SERVICE_ROLE_KEY=your_supabase_service_role_key
+SUPABASE_URL=your_supabase_url
+SUPABASE_SERVICE_ROLE_KEY=your_supabase_service_role_key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SUPABASE_SERVICE_ROLE_KEY key should go before the SUPABASE_URL key

(UnorderedKey)